common/types/block.go

```diff
@@ -191,7 +191,7 @@ func (b *Block) Fields() []log.LoggableField {
 		log.FieldNamed("miner_id", b.MinerID()),
 		log.Int("view_edges", len(b.ViewEdges)),
 		log.Int("vote_count", len(b.BlockVotes)),
-		b.ATXID,
+		log.String("atxid", b.ATXID.ShortString()),
```

Comment:
I don't like this change. If we want to print short ATX IDs that's fine, but then we should do it consistently, namely we should change the behaviour of:
```go
// Field returns a log field. Implements the LoggableField interface.
func (t ATXID) Field() log.Field { return t.Hash32().Field("atx_id") }
```

Perhaps all `Hash32`s should be printed shortly in logs, but even if not, we should be consistent with ATX IDs.